lib/api/apiUtils/object/storeObject.js

```diff
@@ -0,0 +1,81 @@
+import { errors } from 'arsenal';
```

Comment:
Please go ahead and squash, rebase and run an end to end and we can merge this then you can open the other pr.